src/buildHTML.js

```diff
@@ -740,6 +755,10 @@ groupTypes.op = function(group, options, prev) {
         large = true;
     }
 
+    // TODO(cbreeden) Currentl mathop atoms have the assumed structure:
+    //   group<atom>.value === { body, symbol, limits, type }
+    //   this may make it difficult for ops that are not either
+    //   a single symbol or list of text.
```

Comment:
Sorry, I don't remember what my concern was here.  We could probably remove most of these `TODO`.  They were just my notes while thinking of alternatives, I believe.